### PR TITLE
Add checksum calculation so that no full sync is performed by SUT

### PR DESCRIPTION
**Pull request by CorgiMan**

### CODE REVIEW

File: test/test-coordinator.js [hunk 1]
```diff
@@ -245,9 +245,13 @@ TestCoordinator.prototype.callEndpoint = function callEndpoint(endpoint, body, c
...
 
 TestCoordinator.prototype.getMembership = function getMembership() {
+    var sutHost = this.sutHostPort.split(':')[0]
+    var sutPort = this.sutHostPort.split(':')[1]
+
     return this.getFakeNodes().map(function(node) {
         return node.toMemberInfo();
-    }).concat(this.extraMembers);
+    }).concat(this.extraMembers)
+    .concat({host:sutHost, port:sutPort, status:'alive', incarnationNumber:this.sutIncarnationNumber});
```
Comment (Motiejus) [inline test/test-coordinator.js]: I find the SUT membership information to be very useful -- maybe you can put this dictionary to a function in `tc`?

**Comment by Motiejus** [comment]
If I understand correctly, this avoids all unintentional full-syncs by the SUT? If yes, then *awesome job*!  :scream_cat:

**Comment by CorgiMan** [comment]
That's right! 
ringpop-node calculates the checksum a bit differently, but I think we can work around it when we implement the ringpop-node bidirectional full syncs

File: test/test-coordinator.js (hunk 1 above)
Comment (CorgiMan) [inline test/test-coordinator.js]: Good idea

**Pull request merged**